### PR TITLE
Update camera_to_hls

### PR DESCRIPTION
**Pull request by varsill**
* Updates the deps. 
* Adjusts the code to new Membrane Core and new HLS plugin

### CODE REVIEW

File: camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex [hunk 1]
```diff
@@ -1,27 +1,39 @@
...
 
+  alias Membrane.HTTPAdaptiveStream.Manifest
+  alias Membrane.Time
+  import Membrane.ChildrenSpec
+
   @impl true
-  def handle_init(_) do
-    children = [
+  def handle_init(_ctx, _opts) do
+    segment_duration = %Manifest.SegmentDuration{min: Time.seconds(2), target: Time.seconds(5)}
```
Comment (serpent213) [inline camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex:10]: `SegmentDuration` was dropped in https://github.com/membraneframework/membrane_http_adaptive_stream_plugin/commit/1458d37fc43500b9c54992968433c27d51a3dc83 (v0.15.0).

**Comment by serpent213** [comment]
Noticed this pull request after making similar changes myself.

Just learned about Membrane today, works pretty much out of the box. Amazing product! Reminds me of the early `erlyvideo` days.  🙂

File: camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex (hunk 1 above)
Comment (varsill) [reply camera_to_hls/lib/membrane_demo_camera_to_hls/pipeline.ex:10]: Indeed, good spot!
Unfortunately this PR has been abandoned so I am closing it in favour of #241 and #239, which update all the demos to Core v0.12 (including `camera_to_hls`).

**Comment by varsill** [comment]
Closing in favour of #239  and #241

**Pull request closed without merging**